src/core/router/router.blueprint.tsx

```diff
@@ -172,6 +173,19 @@ export const blueprint: RouteObject[] = [
           },
         ],
       },
+      {
+        path: 'user/contracts',
+        loader: async () => {
+          const offers = await userOffers({ page: 1, limit: 5 });
+          return offers;
+        },
+        async lazy() {
+          const { UserContracts } = await import('src/Nowruz/pages/userContracts');
```

Comment:
Same here can we just name it contracts instead of userContracts so it is not tied to the type?